src/app/client/src/app/modules/core/services/search/search.service.ts

```diff
@@ -268,6 +268,29 @@ export class SearchService {
     }
     return this.publicDataService.post(option);
   }
+
+  /**
+   * Content Search using dail code.
+   *
+   * @param {SearchParam} requestParam api request data
+  */
+ contentDialCodeAssembleSearch(requestParam: SearchParam, addDefaultContentTypesInRequest: boolean = true):
+ Observable<ServerResponse> {
+ const option = {
+   url: this.config.urlConFig.URLS.DAIL_ASSEMBLE_PREFIX,
+   data: {
+     request: {
+      source: 'web',
+      name: 'DIAL Code Consumption',
```

Comment:
give name as DialCode while creating new page api

Reply:
I don't think so as I discussed with KP team, they said we can't give a custom name. 
This is `name: 'DIAL Code Consumption',` already pre-defined in all ENV's